Tests/FunctionalTests/Edge+ConsentTests.swift

```diff
@@ -137,15 +137,15 @@ class EdgeConsentTests: TestBase, AnyCodableAsserts {
         fireManyEvents()
 
         // verify
-        var resultNetworkRequests = self.mockNetworkService.getNetworkRequestsWith(url: TestConstants.EX_EDGE_INTERACT_PROD_URL_STR, httpMethod: HttpMethod.post, expectationTimeout: 2)
+        var resultNetworkRequests = self.mockNetworkService.getNetworkRequestsWith(url: TestConstants.EX_EDGE_INTERACT_PROD_URL_STR, httpMethod: HttpMethod.post, timeout: 2)
```

Comment:
Extract timeout as a constant and standardize timeouts. I am seeing mix of 1 sec and 2 sec, we can make it same.